src/Apps/Artwork/routes.tsx

```diff
@@ -1,5 +1,7 @@
 import loadable from "@loadable/component"
+import { request } from "graphql-request"
```

Comment:
I added this lib just to test, we can use isomorphic fetch so new deps aren't added